tests/zenko_tests/python_tests/zenko_e2e/grafana/client.py

```diff
@@ -8,20 +10,36 @@ def __init__(self, user, password, endpoint):
         self._password = password
         self._endpoint = endpoint
 
+    def _get(self, path):
+        backoff = 2
+        for _ in range(4):  # One try with 3 retries
+            try:
+                resp = requests.get(
+                    path,
+                    auth=HTTPBasicAuth(self._user, self._password)
+                )
+                assert resp
+                assert resp.status_code == 200
+                assert resp.json()
+                return True
```

Comment:
Don't you want to `return resp.json()` not to change the previous behavior?

Reply:
It's only the grafana tests that use this so I think a  little rework is OK, also it keeps our requests checking in one place rather that duplicating it